### PR TITLE
[WIP]Change output file to hdfs

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-graphene/README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ The result should look like: <br>
 >   ############# model saved[P1182:T2:java] ---- end time: 1985297 ms return from shim_write(...) = 0x19 <br>
 
 ##### Example Test 3
-Before run TPC-H test in container we created, we should download and install [SBT](https://www.scala-sbt.org/download.html), then build and package TPC-H dataset according to [TPC-H](https://github.com/qiuxin2012/tpch-spark) with your needs. After packaged, check if we have `spark-tpc-h-queries_2.11-1.0.jar ` under `/tpch-spark/target/scala-2.11`, if have, we package successfully.
+Before run TPC-H test in container we created, we should download and install [SBT](https://www.scala-sbt.org/download.html) and [HDFS](https://hadoop.apache.org/docs/r1.2.1/), then build and package TPC-H dataset according to [TPC-H](https://github.com/qiuxin2012/tpch-spark) with your needs. After packaged, check if we have `spark-tpc-h-queries_2.11-1.0.jar ` under `/tpch-spark/target/scala-2.11`, if have, we package successfully.
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/scala/docker-graphene/README.md:117]: and deploy a hdfs

Comment (glorysdj) [inline ppml/trusted-big-data-ml/scala/docker-graphene/README.md:117]: and deploy a hdfs for tpch output?

File: ppml/trusted-big-data-ml/scala/docker-graphene/README.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ SGX=1 ./pal_loader /opt/jdk8/bin/java \
         --total-executor-cores 4 \
         --executor-memory 10G \
         /ppml/trusted-big-data-ml/work/tpch-spark/target/scala-2.11/spark-tpc-h-queries_2.11-1.0.jar \
-        /ppml/trusted-big-data-ml/work/tpch-spark/dbgen | tee spark.local.tpc.h.sgx.log
+        hdfs://LOCAL_IP:8020/dbgen hdfs://LOCAL_IP:8020/tmp/output | tee spark.local.tpc.h.sgx.log
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/scala/docker-graphene/README.md:155]: LOCAL_IP to HDFS_NAMENODE_IP?

Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-graphene/README.md]: tks,done~

File: ppml/trusted-big-data-ml/scala/docker-graphene/README.md (hunk 1 above)
Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-graphene/README.md]: Thanks, done~

**Pull request merged**